src/plugins/maps_ems/common/index.ts

```diff
@@ -6,16 +6,18 @@
  * Side Public License, v 1.
  */
 
-export const TMS_IN_YML_ID = 'TMS in config/kibana.yml';
+export {
+  DEFAULT_EMS_FILE_API_URL,
+  DEFAULT_EMS_TILE_API_URL,
+  DEFAULT_EMS_LANDING_PAGE_URL,
+  DEFAULT_EMS_FONT_LIBRARY_URL,
+  DEFAULT_EMS_ROADMAP_ID,
+  DEFAULT_EMS_ROADMAP_DESATURATED_ID,
+  DEFAULT_EMS_DARKMAP_ID,
+  EMS_APP_NAME,
+} from './ems_defaults';
 
-export const DEFAULT_EMS_FILE_API_URL = 'https://vector.maps.elastic.co';
-export const DEFAULT_EMS_TILE_API_URL = 'https://tiles.maps.elastic.co';
-export const DEFAULT_EMS_LANDING_PAGE_URL = 'https://maps.elastic.co/v8.0';
-export const DEFAULT_EMS_FONT_LIBRARY_URL =
-  'https://tiles.maps.elastic.co/fonts/{fontstack}/{range}.pbf';
+export { EMSSettings } from './ems_settings';
+export type { EMSConfig } from './ems_settings';
 
-export const DEFAULT_EMS_ROADMAP_ID = 'road_map';
-export const DEFAULT_EMS_ROADMAP_DESATURATED_ID = 'road_map_desaturated';
-export const DEFAULT_EMS_DARKMAP_ID = 'dark_map';
-
-export { ORIGIN } from './origin';
+export const MAPS_EMS_APP_ID = 'maps';
```

Comment:
Maybe the name MAPS_EMS_APP_ID is not correct because the plugin ID is not 'maps' but really 'mapsEms'. how about renaming to something more inline with usage like LICENSE_CHECK_ID?